Added an option to pass connection params when initiating connection … (#3403)
strawberry-graphql/strawberry: RELEASE.md

Release type: minor

> Added support to pass connection_params as dictionary to GraphQLWebsocketCommunicator when testing [channels integration](https://strawberry.rocks/docs/integrations/channels#testing)

Reviewer: ```suggestion
This release adds support to allow passing `connection_params` as dictionary to `GraphQLWebsocketCommunicator` class when testing [channels integration](https://strawberry.rocks/docs/integrations/channels#testing)
```

Author: updated RELEASE.md file

> ### sample code snippet

Reviewer: ```suggestion
### Example
```

> ```
> GraphQLWebsocketCommunicator(
>     application=application,
>     path="/graphql",
>     connection_params={"username": "strawberry"}
> )
> ```

Reviewer: ````suggestion
```python
GraphQLWebsocketCommunicator(
    application=application,
    path="/graphql",
    connection_params={"username": "strawberry"}
)
```
````

Author: updated RELEASE.md file
